core/ui/compose/designsystem/src/test/kotlin/app/k9mail/core/ui/compose/designsystem/atom/textfield/TextInputTextFieldTest.kt

```diff
@@ -49,6 +56,23 @@ class TextInputTextFieldTest(
         assertThat(value).isEqualTo("$testInput + added text")
     }
 
+    @Test
+    fun `should strip line breaks before onValueChange is called`() = runComposeTest {
```

Comment:
This is only true when `isSingleLine` is enabled. With this test `TextFieldOutlined` is forces to always strip linebreaks, which is undesired for multiline text input. You added `TextFieldOutlinedKtTest` to compensate for that, but I think using a parameterized test reached a state where it's more a burden than helpful.

Reply:
I deleted `TextInputTextFieldTest` and created test classes for the individual composables.